src/Utilities/Compiler/Options/EditorConfigOptionNames.cs

```diff
@@ -28,6 +28,11 @@ internal static partial class EditorConfigOptionNames
         /// </summary>
         public const string ExcludeAsyncVoidMethods = "exclude_async_void_methods";
 
+        /// <summary>
+        /// Boolean option to enable platform commpatibility analyzer for lower version than 5.0.
```

Comment:
Please add a section at the end of https://github.com/dotnet/roslyn-analyzers/blob/master/docs/Analyzer%20Configuration.md#supported-editorconfig-options for the new option.